translator/tests/acceptance/steps/actions.py

```diff
@@ -4,16 +4,18 @@
 from behave import then, when
 
 
-@when("we add {route} to the block list")
-def add_block(context, route):
+@when("we add {route} with {asn} and {community} to the block list")
+def add_block(context, route, asn, community):
     ip = ipaddress.ip_interface(route)
-    context.gobgp.add_path(ip)
+    event_data = {"asn": int(asn), "community": int(community)}
+    context.gobgp.add_path(ip, event_data)
 
 
 @when("we delete {route} from the block list")
 def del_block(context, route):
     ip = ipaddress.ip_interface(route)
-    context.gobgp.del_path(ip)
+    event_data = {"asn": 64500, "community": 666}
```

Comment:
We need to pass these as vars, otherwise we'll get a mismatch if we don't use the default ones.